bootstrap/api/v1beta2/ck8sconfig_types.go

```diff
@@ -51,6 +51,20 @@ type CK8sConfigSpec struct {
 	// +optional
 	AirGapped bool `json:"airGapped,omitempty"`
 
+	// Track is the channel to use for the snap install.
+	// +optional
+	Track string `json:"track,omitempty"`
```

Comment:
```suggestion
	Channel string `json:"channel,omitempty"`
```

Let's rename to channel, track is just a [part of the channel.](https://snapcraft.io/docs/channels)